src/pages/MyProfile.module.css

```diff
@@ -0,0 +1,15 @@
+.wrapper {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+}
+
+.missions,
```

Comment:
On the `Missions` page you have some styling issues. Your `leave mission` button is becoming unreadable to a user. Kindly change its style to make it more visible and readable.

<img width="144" alt="Screen Shot 2022-08-26 at 06 11 14" src="https://user-images.githubusercontent.com/69011963/186796830-887b1423-0df4-4d03-958e-d5faa5a7a9ea.png">